ts/components/messages/MessageAgenda.tsx

```diff
@@ -133,6 +155,9 @@ type OwnProps = {
 type Props = OwnProps & SelectedSectionListProps;
 
 type State = {
+  isAnimStarted: boolean;
```

Comment:
@Edigio 
instead of using 2 variables
```
isAnimStarted: boolean;
readyToStartAnim: boolean;
```
tou can try to use **pot**
I suppose it is right representing this kind of situation
```
pot.none -> hide button
pot.loading -> animation can be started
pot.some -> animation is running
```

Reply:
@Undermaken i used POT instead Booleans, is my implementation correct?